RoslynPluginGenerator/AnalyzerPluginGenerator.cs

```diff
@@ -21,6 +21,12 @@ namespace SonarQube.Plugins.Roslyn
 {
     public class AnalyzerPluginGenerator
     {
+        /// <summary>
+        /// Specifies the format for the name of the placeholder SQALE file
+        /// </summary>
+        public const string SqaleTemplateFileNameFormat = "{0}.{1}.sqale.template.xml";
```

Comment:
The template file name will include package id and version so they are package-specific (in case the user runs the analyzer multiple times against different packages).